README.md

```diff
@@ -2,51 +2,34 @@
 
 File programming language detector and toolbox to ignore binary or vendored files. *enry*, started as a port to _Go_ of the original [linguist](https://github.com/github/linguist) _Ruby_ library, that has an improved *2x performance*.
```

Comment:
Question: Does the 2x performance improvement include using Oniguruma?

Reply:
No, all the benchmarks supporting this performance claim are done without Oniguruma, which always was regarded as an optional improvement.